src/twisted/web/_newclient.py

```diff
@@ -797,6 +797,15 @@ def f(passthrough):
         return d
 
 
+    def _writeToEmptyBodyContentLength(self, transport):
+        """
+        Write this request to the given transport using content-length to frame
```

Comment:
`twistedchecker` complains that `transport` isn't documented, nor is the return value.